clusters/clusters_api.go

```diff
@@ -399,7 +399,7 @@ type Cluster struct {
 	ClusterID   string `json:"cluster_id,omitempty"`
 	ClusterName string `json:"cluster_name,omitempty"`
 
-	SparkVersion              string     `json:"spark_version"`
+	SparkVersion              string     `json:"spark_version,omitempty"`
```

Comment:
Without this change, the jobs unit tests failed with:
```
panic: new_cluster: inconsistency: spark_version is optional, default is empty, but has no omitempty
```

Reflection on the 2.0 structs flagged the incompatibility between an optional field in the schema without a default and a mandatory struct field. This struct is no longer used directly by a resource.